test/tuntap-integration.spec.js

```diff
@@ -0,0 +1,132 @@
+import assert from "assert";
+import { TunTap } from "../lib/index.js";
+import { spawn } from "child_process";
+import path from "path";
+import { fileURLToPath } from "url";
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+describe("TunTap Integration Tests", function () {
+  let tun;
+
+  before(function () {
+    if (typeof process.getuid === "function" && process.getuid() !== 0) {
+      this.skip();
```

Comment:
it is also possible to provide a skip reason as an argument

Reply:
updated with commit https://github.com/appium/appium-ios-tuntap/pull/1/commits/bcb9e8623f1bc9c54f8dfeb8e9c645d58dcfad4d